### DISCUSSION TITLE
Edit the Current Ticket

### DISCUSSION THREAD

**Discussion by engahmadkadi** [ENDER-9/Vocascan#103] [Ideas]
Hi vocascan,

I wish to introduce the feature of editing the shown ticket directly while learning, by showing an edit button on the corner of both sides of the ticket. This eases the continuous enhancement of own vocabularies.

Thanks a lot!

**Comment by luwol03** [comment]
Hey engahmadkadi,

welcome to vocascan, glad you found vocascan and are eager to suggest new ideas and features.

Editing cards while learning them is actually a great idea we never thought of. (However I know that situation of noticing some mistakes while learning)

Can you please open a new issue to track that request? Discussions are more like to ask questions or showing things people have done with vocascan.

luwol03

**Comment by luwol03** [comment]
Hey engahmadkadi,

as I already said, we are glad you found vocascan and you are eager to suggest new ideas and features. I don't know how to ask and if I can ask that question, but we are interested in where did you find us? (On awesome-selfhosted, the website, discord, github, Twitter, ...)

luwol03

**Reply by engahmadkadi** [reply]
I'm also glad to have found this tool. 

What I did is only search for `own vocabulary trainer` using a search engine of github and saw the repos and landed on your website and documentations. This was sometime ago (one month probably), but I hesitated to create an account on the server directly. I didn't want to put my files on server and create account. But the nice and organized documentation impressed me, so I tried to give it a chance by installing it on may mac. Unfortunately I wasn't able to have it running (link server and client docker containers), so I gave up too fast.

Later in the time, I searched for same terms (as the need of such tools keep coming to mind) to encounter vocascan again. This time I dared to create an account and played around with the tools and find it meeting my needs.

I'm looking forward to be part in carrying the project to further success even with my limited knowledge.

Ahmad

**Reply by noctera** [reply]
I am glad to hear that, thanks for the praise.

As you said, you had some problems setting up a Vocascan server with docker... Maybe you still remember what your problem was, so we could help you and adjust the documentation if necessary.

However we try to keep the greatest privacy for the Vocascan cloud. We store only the most necessary personal data. And even those could not be accessed by anyone. The only personal data that is stored in our database, email and password, is hashed. So they can not be seen by anyone

noctera

**Reply by engahmadkadi** [reply]
Great to hear that.

I'll come back soon to you once I get the chance to try the installation again :)

Ahmad

**Reply by luwol03** [reply]
In addition to what noctera said, we also planned but not fully decided to ship a simple server with the electron client, so you can learn your vocabularies only with the desktop app and no 3rd party server will be involved.
